diskqueue.go

```diff
@@ -101,16 +104,39 @@ type diskQueue struct {
 	exitSyncChan      chan int
 
 	logf AppLogFunc
+
+	// disk limit implementation flag
+	diskLimitFeatIsOn bool
```

Comment:
Feat short for feature can be confusing here. enableDiskLimitation?

Reply:
Great suggestion! I will change the variable name